src/client/apps/edit/components/content/article_layouts/series.jsx

```diff
@@ -64,12 +65,15 @@ export class EditSeries extends Component {
     const description = article.series && article.series.description
 
     return (
-      <Paragraph
-        html={description || ''}
-        linked
-        onChange={(html) => onChangeArticleAction('series.description', html)}
-        placeholder='Start writing here...'
-      />
+      <Text layout={article.layout}>
```

Comment:
My idea was to abstract this component so it is no longer dependent on the concept of articles, so that you can just tell the draft component what styles you need.  Was also planning to move this into Reaction so it could be used outside of an article (it is already used in Positron outside of articles). `Text` only wraps the component in styling that is particular to an article layout.